validation/util/linux_resources_devices.go

```diff
@@ -48,6 +45,5 @@ func ValidateLinuxResourcesDevices(config *rspec.Spec, state *rspec.State) error
 		}
 	}
 
-	t.AutoPlan()
```

Comment:
According to your idea, there is `t.AutoPlan()` in this file that has not been deleted yet.

Reply:
@q384566678 Good catch.
I've removed `t.AutoPlan()` from `linux_resources_devices.go` as well as `linux_resources_blkio.go`.